build.sbt

```diff
@@ -1,13 +1,33 @@
 // Copyright: 2017 https://github.com/cakesolutions/sbt-cake/graphs
 // License: http://www.apache.org/licenses/LICENSE-2.0
 
+import de.heikoseeberger.sbtheader.FileType
+import de.heikoseeberger.sbtheader.HeaderPlugin.autoImport.HeaderFileType
+
 organization := "net.cakesolutions"
 
+// scalastyle:off magic.number
+
 name := "sbt-cake"
 sbtPlugin := true
 
 sonatypeGithub := ("cakesolutions", "sbt-cake")
+startYear := Some(2017)
 licenses := Seq(Apache2)
+headerLicense := Some(
+  HeaderLicense.Custom(
+    """|Copyright: 2017 https://github.com/cakesolutions/sbt-cake/graphs
```

Comment:
When I tried this locally, I didn't get this type of copyright header being generated. Hence why I opted for an explicit approach.